CONTRIBUTING.md

```diff
@@ -45,7 +45,7 @@ by discussion, compromise on the proposed change, or withdrawal of the proposed
 
 When making a bug fix contribution, please add the [bug](https://github.com/USGS-Astrogeology/ISIS3/pulls?q=is%3Apr+label%3Abug+) label to your pull request. This will automatically cherry-pick your PR into the latest release branch so that it can be distributed in the next bug fix release. 
 
-Every pull request making changes to ISIS's source needs a [changelog entry](https://raw.githubusercontent.com/DOI-USGS/ISIS3/dev/CHANGELOG.md). PRs are limited to one changelog entry per PR. If you have multiple bug fixes or feature adds, they should be split up when possible. If PRs need to be merged in a particular order, include that as a comment on your PRs. If PR is one changelog, but high-volume changes such as large feature branches (many changes across many files requiring several hours of review time), then the PR isn't guaranteed to be reviewed promptly. Authors should contact the repo owners, see [code.json](https://github.com/DOI-USGS/ISIS3/blob/dev/code.json), to coordinate reviews outside typical USGS Astro support hours, preferably ahead of time.
+Every pull request making changes to ISIS's source needs a [changelog entry](https://raw.githubusercontent.com/DOI-USGS/ISIS3/dev/CHANGELOG.md). PRs are limited to one changelog entry per PR. If you have multiple bug fixes or feature adds, they should be split up. If PRs need to be merged in a particular order, include that as a comment on your PRs. If PR is one changelog, but high-volume changes such as large feature branches (many changes across many files requiring several hours of review time), then the PR isn't guaranteed to be reviewed promptly. Authors should contact the repo owners, see [code.json](https://github.com/DOI-USGS/ISIS3/blob/dev/code.json), to coordinate reviews outside typical USGS Astro support hours, preferably ahead of time.
```

Comment:
"preferably ahead of time"

How much lead time should authors expect to account for? Depending on when our scheduling meeting occurred, our schedules might be tied to specific projects for 4-6 weeks.  Is this all just part of the coordination / negotiation that will occur between authors/owners on a case-by-base basis, or do we want to establish a more general rule?

Reply:
It would case-by-case. AFAIK we haven't had an official process for this as it seems its specific people negotiating for specific teams. Using scheduling as a time frame makes sense, but some of it also includes negotiating the time and costs so I don't think there could be a hard and fast rule.

If we mention a specific time, maybe setting 6 weeks as a minimum makes sense.

Reply:
I think that it makes sense to let it be a case-by-case basis, but it also might make sense to provide a rough estimate just to set the expectation. If some users feel like 1 week ahead of time is enough to be considered "ahead of time," then they might be in for a rude surprise when they find that that's not going to happen. 

Interested to see if anyone else has feedback on this, or if it's a total non-issue then it can probably just be left as written.

Reply:
Personal opinion - the more explicit the expectations are the less likely folks are to be frustrated. Misaligned expectations are hard to work through remotely / asynchronously in my experience.